torch/_decomp/decompositions.py

```diff
@@ -609,6 +610,58 @@ def slice_backward(
     return torch.slice_scatter(grad_input, grad_output, dim, start, end, step)
 
 
+@register_decomposition(aten.slice.Tensor)
+def slice_forward(
```

Comment:
dumb q: why do we need a python decomp for `slice` if we're already `c10::SymInt`-ifying the C++ implementation? Is the python impl still required for something? (either way, I totally see a python impl being useful to have).

Reply:
Look over the patch carefully. We did not, in fact, SymInt'ify the C++ implementation.

Reply:
welp, thanks. Needed to scan the codegen again to remember how it works: If we put `_symint` at the end of the kernel name in `native_functions.yaml`, then the codegen will expect you to symint-ify the C++ kernel